js/curation-tool.js

```diff
@@ -356,7 +356,7 @@ var vm = new Vue({
             if(phylotree === undefined)
                 return "()";
 
-            return phylotree.get_newick(function(n) { return n; });
+            return get_newick_string_from_phylotree(phylotree);
```

Comment:
It looks like we should extend the phylotree object for our purposes? If that's not filed as an issue yet, it should be.